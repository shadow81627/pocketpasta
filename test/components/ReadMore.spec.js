import Component from '@/components/ReadMore.vue';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vue from 'vue';
import Vuetify from 'vuetify';

const localVue = createLocalVue();

Vue.use(Vuetify);

describe('ReadMore', () => {
  let vuetify;
  beforeEach(() => {
    vuetify = new Vuetify();
  });
  const factory = () => shallowMount(Component, { vuetify, localVue });

  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.vm).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
