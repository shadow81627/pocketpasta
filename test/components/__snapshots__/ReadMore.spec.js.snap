// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReadMore renders properly 1`] = `"<span><span></span> <span style=\\"display: none;\\"><a href=\\"#\\">Read more</a> <a href=\\"#\\" style=\\"display: none;\\">Read less</a></span></span>"`;
